src/stores/__tests__/chat_messages_store-test.js

```diff
@@ -16,10 +16,11 @@ describe('ChatMessagesStore', () => {
     callback = Dispatcher.register.mock.calls[0][0];
   });
 
-  describe('CHAT_MESSAGE_RECEIVED', () => {
+  describe('CHAT_SERVER_MESSAGE_RECEIVED', () => {
```

Comment:
Nice work on these tests, by the way, especially with a broken `jest` installation.